google/cloud/accessapproval/doc/main.dox

```diff
@@ -86,7 +86,7 @@ uses [exponential backoff] to backoff between retries. Application developers
 can override the default policies.
 
 [exponential backoff]: https://en.wikipedia.org/wiki/Exponential_backoff
-[gcs-quickstart]: https://cloud.google.com/accessapproval/docs/quickstarts 'Quickstarts'
+[gcs-quickstart]: https://cloud.google.com/access-approval/docs/quickstarts 'Quickstarts'
```

Comment:
link is broken. use:
https://cloud.google.com/access-approval/docs/quickstart-google-key

Reply:
Good catch. I found: https://cloud.google.com/cloud-provider-access-management/access-approval/docs/quickstart which has all the quickstarts.

Reply:
even better.

FYI: the other links omit the `cloud-provider-access-management/`. The links all redirect to the longer url. Your call about updating all or none of the other links.